backend/src/main/java/COMP_49X_our_search/backend/database/entities/Department.java

```diff
@@ -0,0 +1,78 @@
+package COMP_49X_our_search.backend.database.entities;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "departments")
+public class Department {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Integer id;
+
+  @Column(nullable = false)
+  private String name;
+
+  @ManyToMany(mappedBy = "departments")
+  private Set<Faculty> faculties = new HashSet<>();
+
+  @ManyToMany(mappedBy = "departments")
+  private Set<Project> projects = new HashSet<>();
+
+  @OneToMany(mappedBy = "department")
+  private Set<Major> majors = new HashSet<>();
```

Comment:
This entity looks good, but we may want to triple check to make sure that the departments to majors relationship is one to many

Reply:
True, I'll clarify with Dr. Clark and come back to this.

Reply:
Clarified with Dr. Clark, it should be Many-to-Many